Rocket.Chat/Models/Subscription/Subscription.swift

```diff
@@ -133,26 +133,40 @@ final class RoomRoles: Object {
 // MARK: Failed Messages
 
 extension Subscription {
+    func setTemporaryMessagesFailed() {
+        try? realm?.write {
+            messages.filter("temporary = true").forEach {
+                $0.temporary = false
+                $0.failed = true
+            }
+        }
+    }
 
-    func avatarURL(auth: Auth? = nil) -> URL? {
+}
+
+// MARK: Avatar
+
+extension Subscription {
+    static func avatarURL(name: String, auth: Auth? = nil) -> URL? {
         guard
             let auth = auth ?? AuthManager.isAuthenticated(),
             let baseURL = auth.baseURL(),
             let encodedName = name.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlQueryAllowed)
-        else {
-            return nil
+            else {
```

Comment:
Indentation glitch

Reply:
🤦🏻‍♂️ Hope apple makes this the default indentation.